LocalSettings.php

```diff
@@ -201,199 +213,344 @@
+$wgGroupPermissions['user']['translate-messagereview'] = true;
+$wgGroupPermissions['user']['user'] = true;
 $wgGroupPermissions['user']['writeapi'] = false;
-$wgGroupPermissions['bot']['writeapi'] = false;
-$wgGroupPermissions['sysop']['oathauth-enable'] = true;
-$wgGroupPermissions['sysop']['pagetranslation'] = true;
-$wgGroupPermissions['sysop']['translate-manage'] = true;
-$wgGroupPermissions['sysop']['deleterevision'] = true;
-$wgGroupPermissions['sysop']['interwiki'] = false;
-$wgGroupPermissions['sysop']['editinterface'] = false;
-$wgGroupPermissions['sysop']['upload_by_url'] = true;
-$wgGroupPermissions['sysop']['oathauth-disable-for-user'] = false;
-$wgGroupPermissions['sysop']['abusefilter-modify'] = true;
+
+/// Confirmed
+$wgGroupPermissions['confirmed']['autoconfirmed'] = true;
+$wgGroupPermissions['confirmed']['editsemiprotected'] = true;
+$wgGroupPermissions['confirmed']['movefile'] = true;
+$wgGroupPermissions['confirmed']['move'] = true;
+$wgGroupPermissions['confirmed']['move-categorypages'] = true;
+$wgGroupPermissions['confirmed']['move-rootuserpages'] = true; // can move root userpages
+$wgGroupPermissions['confirmed']['move-subpages'] = true;
+$wgGroupPermissions['confirmed']['skipcaptcha'] = true;
+
+/// Auto confirmed users
+$wgGroupPermissions['autoconfirmed']['movefile'] = true;
+$wgGroupPermissions['autoconfirmed']['move'] = true;
+$wgGroupPermissions['autoconfirmed']['move-categorypages'] = true;
+$wgGroupPermissions['autoconfirmed']['move-rootuserpages'] = true; // can move root userpages
+$wgGroupPermissions['autoconfirmed']['move-subpages'] = true;
+$wgGroupPermissions['autoconfirmed']['skipcaptcha'] = true;
+
+/// Auto patrollers
+$wgGroupPermissions['autopatrol']['autopatrol'] = true;
+
+/// Patrollers
+$wgGroupPermissions['patroller']['autopatrol'] = true;
+$wgGroupPermissions['patroller']['noratelimit'] = true;
+$wgGroupPermissions['patroller']['patrol'] = true;
+$wgGroupPermissions['patroller']['rollback'] = true;
+$wgGroupPermissions['patroller']['suppressredirect'] = true;
+$wgGroupPermissions['patroller']['upload_by_url'] = true;
+
+/// Translation administrators
+$wgGroupPermissions['translateadmin']['pagelang'] = true;
+$wgGroupPermissions['translateadmin']['pagetranslation'] = true;
+$wgGroupPermissions['translateadmin']['translate-manage'] = true;
+
+/// Chat moderators
+$wgGroupPermissions['chatmod']['chat'] = false;
+
+/// Interwiki administrators
+$wgGroupPermissions['interwiki-admin']['interwiki'] = true;
+
+/// Protecter
+$wgGroupPermissions['protecter']['protect'] = true;
+$wgGroupPermissions['protecter']['editprotected'] = true;
+$wgGroupPermissions['protecter']['editsemiprotected'] = true;
+
+/// Deleter
+$wgGroupPermissions['deleter']['delete'] = true;
+$wgGroupPermissions['deleter']['bigdelete'] = true;
+$wgGroupPermissions['deleter']['deleterevision'] = true;
+$wgGroupPermissions['deleter']['deletechangetags'] = true;
+
+/// Abusefilter restricted users
```

Comment:
this seems stripped down from what it currently is?

Reply:
Like mine?